contracts/strategies/ltip-simple/LTIPSimpleStrategy.sol

```diff
@@ -0,0 +1,708 @@
+            msg.sender
+        );
+    }
+
+    /// ====================================
+    /// ============ Internal ==============
+    /// ====================================
+
+    /// @notice Submit a proposal to LTIP pool
+    /// @dev Emits a 'Registered()' event
+    /// @param _data The data to be decoded
+    ///     - If registryGating is true, then the data is encoded as (address recipientId, address recipientAddress, uint256 allocationAmount, Metadata metadata)
+    ///     - If registryGating is false, then the data is encoded as (address recipientAddress, address registryAnchor, uitn256, allocationAmount, Metadata metadata)
+    /// @custom:data-registry (address recipientId, address recipientAddress, uint256 allocationAmount, Metadata metadata)
+    /// @custom:data-no-registry (address recipientAddress, address registryAnchor, uint256 allocationAmount, Metadata metadata)
+    /// @param _sender The sender of the transaction
+    /// @return recipientId The id of the recipient
+    function _registerRecipient(bytes memory _data, address _sender)
+        internal
+        override
+        onlyActiveRegistration
+        returns (address recipientId)
+    {
+        bool isUsingRegistryAnchor;
+        address recipientAddress;
+        address registryAnchor;
+        uint256 allocationAmount;
+        Metadata memory metadata;
+
+        // decode data custom to this strategy
+        if (registryGating) {
+            (recipientId, recipientAddress, allocationAmount, metadata) =
+                abi.decode(_data, (address, address, uint256, Metadata));
+
+            // when registry gating is enabled, the recipientId must be a profile member
+            if (!_isProfileMember(recipientId, _sender)) revert UNAUTHORIZED();
+        } else {
+            (recipientAddress, registryAnchor, allocationAmount, metadata) =
+                abi.decode(_data, (address, address, uint256, Metadata));
+            isUsingRegistryAnchor = registryAnchor != address(0);
+            recipientId = isUsingRegistryAnchor ? registryAnchor : _sender;
+
+            // when using registry anchor, the ID of the recipient must be a profile member
+            if (isUsingRegistryAnchor && !_isProfileMember(recipientId, _sender)) revert UNAUTHORIZED();
+        }
+
+        // Check if the metadata is required and if it is, check if it is valid, otherwise revert
+        if (metadataRequired && (bytes(metadata.pointer).length == 0 || metadata.protocol == 0)) {
+            revert INVALID_METADATA();
+        }
+
+        // If the recipient address is the zero address this will revert
+        if (recipientAddress == address(0)) revert RECIPIENT_ERROR(recipientId);
+
+        // Get the recipient
+        Recipient storage recipient = _recipients[recipientId];
+
+        // update the recipients data
+        recipient.recipientAddress = recipientAddress;
+        recipient.useRegistryAnchor = registryGating ? true : isUsingRegistryAnchor;
```

Comment:
`recipient.useRegistryAnchor` is not really being used anywhere, I think it makes sense to remove that property from the `Recipient` struct too.